arch/risc-v/src/esp32c3/esp32c3_tim.c

```diff
@@ -925,6 +929,11 @@ struct esp32c3_tim_dev_s *esp32c3_tim_init(int timer)
       case 1:
         {
           tim = &g_esp32c3_tim1_priv;
+
+          modifyreg32(SYSTEM_PERIP_CLK_EN0_REG, 0,
+                      SYSTEM_TIMERGROUP1_CLK_EN);
```

Comment:
I think you can keep it in a single line

Reply:
No, it exceeds the line limit of 78 chars.